client/ui/profile.js

```diff
@@ -1,20 +1,26 @@
-const WEAPON_PATH = "/assets/gfx/";
+const GFX_PATH = "/assets/gfx/";
 const WEAPONS = {
   bow: "bow.png",
   sword: "sword.png",
 };
+const ITEMS = {
+  wood: "wood.png",
+  "copper ore": "copper-ore.png",
```

Comment:
Is this space here safe? Maybe `copper_ore` instead?

Reply:
It is safe
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors